mypy/main.py

```diff
@@ -203,6 +204,16 @@ def add_invertible_flag(flag: str,
             strict_flag_names.append(flag)
             strict_flag_assignments.append((dest, not default))
 
+    def disallow_any_argument_type(raw_options: str) -> List[str]:
+        current_options = raw_options.split(',')
+        for option in current_options:
+            if option not in valid_disallow_any_options:
+                formatted_opts = ', '.join(map("'{}'".format, valid_disallow_any_options))
```

Comment:
I'd also consider calling this "formatted_valid_options" in order to disambiguate from the other "*_options" variables.  I'd also write out "options" fully here -- it's a bit more verbose, but is more consistent with the rest of the variable names.